Longest_Abecedarian_Word/Program.cs

```diff
@@ -4,9 +4,54 @@ namespace Longest_Abecedarian_Word
 {
     class Program
     {
+
+        public static string LongestAbecedarian(string[] array)
+        {
+            string Alphabet = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
Use var instead strict variable type declaration when it possible.
var Alphabet = "abcdefghijklmnopqrstuvwxyz";